homework3.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+	"unicode/utf8"
+)
+
+/*Task 1 : average([...]int) float64 function that returns
+an average value of array (sum / N)*/
+func average(array []int) float64 {
+	if array == nil {
+		return 0
+	}
+	var sum int
+	for _, item := range array {
+		sum += item
+	}
+	return float64(sum) / float64(len(array))
+}
+
+/*Task 2.1 : function that returns the longest word from the slice
+of strings (the first if there are more than one).*/
+func max(array []string) string {
+	if array == nil {
```

Comment:
Will be better to check the length of slice here.

Reply:
Edited.